addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -43,6 +43,7 @@
 import org.apache.commons.lang.StringUtils;
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
+import org.jsoup.Jsoup;
```

Comment:
@mgeramb Can you rebase your branch? We agreed in #4838 that Jsoup is not required. Or did it change in this PR?

Reply:
No, this was not what I want. Sorry, I will check this.

Reply:
Oh, it is now again required for parsing the SSML. But this is an XML not HTML format, so I could use an XML parser. Which one can I use for this? I just need the text without all the xml nodes.
```
if (text.startsWith("<speak>") && text.endsWith("</speak>")) {
            content.speak.type = "ssml";
            String plainText = Jsoup.parse(text).text();
            content.display.body = plainText;
        }
```

Reply:
Following packages are available by default for XML processing:

- comm.thoughtworks.xstream
- com.thoughtworks.xstream.annotations
- com.thoughtworks.xstream.converters
- com.thoughtworks.xstream.io
- com.thoughtworks.xstream.io.xml

See http://x-stream.github.io/

Reply:
Ok, I does not find the required function in this library, but I have changed the code to use a simple regex for this case.